platform/ext/target/nordic_nrf/nrf5340/tfm_peripherals_def.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018-2019, Arm Limited. All rights reserved.
+ * Copyright (c) 2020, Cypress Semiconductor Corporation. All rights reserved.
+ * Copyright (c) 2020, Nordic Semiconductor ASA. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ */
+
+#ifndef __TFM_PERIPHERALS_DEF_H__
+#define __TFM_PERIPHERALS_DEF_H__
+
+#include "platform_irq.h"
```

Comment:
We haven't added such file yet. And I think we should not do it, we should include here `<nrf.h>` instead to get `TIMER0_IRQn` and `TIMER1_IRQn` defined.

Reply:
Yeap, correct. This is related to my question few days back about introducing a dummy platform_irq.h header, if you remember. 

Will remove

Reply:
Fixed now